src/Energon.php

```diff
@@ -44,26 +44,48 @@ public static function generate($prime = null)
     }
 
     /**
+     * Generates a set of numbers ready for use (with a custom max cap).
+     *
+     * @param int $max
+     *
+     * @return array
+     */
+    public static function generateMax($max)
```

Comment:
Can we remove this method and add a parameter to the existing `generate` method?